clients/drcachesim/tests/scheduler_unit_tests.cpp

```diff
@@ -5612,6 +5612,114 @@ test_unscheduled_initially_roi()
 #endif
 }
 
+static void
+test_unscheduled_initially_rebalance()
+{
+    // Tests i#7318 where on a rebalance a too-large runqueue has nothing
```

Comment:
s/rebalance/rebalance attempt/

Reply:
Done, though as is sounds fine to me with "rebalance" being a noun on its own.